### ISSUE TITLE
Error: provided pretrained model only used in single-sample binning!

### ISSUE THREAD

**Issue by yazhinia** [BigDataBiology/SemiBin#172]
Hi,
   I am using `SemiBin2 single_easy_bin` command with pretrained model and 10 samples. I get an error that provided pretained model (global or ocean) only used in single-sample binning!.

Is it expected?

Thank you.

Best,
Yazhini

**Comment by luispedro** [comment]
Yes, as of now, pretrained models can only be used for single-sample binning. This is because the model depends on the number of samples and we'd have to have a pretrained model for _10_ samples (and another for _11_ samples and another for _12_ sample, &c)

**Comment by yazhinia** [comment]
Thank you for the reply. So If I understand correctly that I have to run SemiBin(2) with training mode if I have more than 1 samples for the dataset.

**Comment by luispedro** [comment]
Yes, that is a current limitation
